Cargo.toml

```diff
@@ -62,7 +62,8 @@ rayon = { version = "1.8", optional = true }
 dbpnoise = { version = "0.1.2", optional = true }
 pathfinding = { version = "4.4", optional = true }
 num-integer = { version = "0.1.45", optional = true }
-dmi = { version = "0.3.1", optional = true }
+dmi = { version = "0.3.4", optional = true }
+tracy_full = { version = "1.6.1", optional = true }
```

Comment:
it's still here

Reply:
also failing build lol

Reply:
There's zone and frame definitions. It's useful to have around, it does noop when features doesn't include enabled since it's just macros.